include/wx/private/threadinfo.h

```diff
@@ -47,12 +41,12 @@ class wxThreadSpecificInfo
     //     logging
     bool loggingDisabled;
 
-#if wxUSE_INTL
-    // Storage for wxTranslations::GetUntranslatedString()
-    wxLocaleUntranslatedStrings untranslatedStrings;
-#endif
-
 #if wxUSE_THREADS
+    // Initialize the wxThreadSpecificInfo.
+    // If this is not called from a thread, the wxThreadInfo will remain NULL
+    // in that thread's context. Use ThreadCleanUp to free when thread ends.
+    static void ThreadInitialize();
```

Comment:
It would make sense to have a `wxThreadInfoInitializer` class calling this in its ctor and `CleanUp()` in its dtor now that we have both functions.